include/IECoreGL/TextureLoader.h

```diff
@@ -62,7 +62,7 @@ class IECOREGL_API TextureLoader : public IECore::RefCounted
 
 		TextureLoader( const IECore::SearchPath &searchPaths );
 
-		TexturePtr load( const std::string &name );
+		TexturePtr load( const std::string &name, const IECore::IntData *maximumResolution = nullptr );
```

Comment:
`IntData` seems like overkill here? Anything wrong with `int`? We could just use `numeric_limits<int>::max` as the default.

Should we be concerned about the breaking change at this juncture? I don't mind for Gaffer's purposes, but it could cause problems at IE and elsewhere perhaps? If so, we could just add an overload with the extra argument to maintain compatibility, and have the old one forward to the new one.

Reply:
Addressed in 2ae7040 and 164a72a. Sorry, kept them separate to make squashing easier later.